src/stacks/index.ts

```diff
@@ -8,3 +8,4 @@ export * from './nestedRDS';
 export * from './nestedEKS';
 export * from './monitoring';
 export * from './nestedAppSync';
+export * from './nestedSQS'
```

Comment:
Missing semicolon

Reply:
... but because linter doesn't fail this, I don't see a problem besides having this line changed in some non-related future commit.